src/ui/react/src/components/base/button-style.js

```diff
@@ -11,11 +11,15 @@
         // Allows validating props being passed to the component.
         propTypes: {
             /**
-             * The style the button should handle as described by http://docs.ckeditor.com/#!/api/CKEDITOR.style
+             * The style the button should handle. It can be a style object as described by http://docs.ckeditor.com/#!/api/CKEDITOR.style
+             * or a string pointing to an object inside the editor instance configuration
```

Comment:
Hey Chema,

This sentence "a string pointing to an object inside the editor instance configuration" will need an example. We shall document this for 1.2.3 release and I'm not sure what exactly could be this string.

Could you please add an example to this comment?